src/OrchardCore.Modules/OrchardCore.Seo/OrchardCore.Seo.csproj

```diff
@@ -15,6 +15,8 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.Apis.GraphQL.Abstractions\OrchardCore.Apis.GraphQL.Abstractions.csproj" />
+    <ProjectReference Include="..\..\OrchardCore\OrchardCore.ContentManagement.GraphQL\OrchardCore.ContentManagement.GraphQL.csproj" />
```

Comment:
Is this one required? Abstractions is not sufficient?

Reply:
Needed it for some calls in the GraphQL Startup, now it's not needed anymore so I removed the dependency again